frontend/components/Layouts/DashboardLayout.tsx

```diff
@@ -1,10 +1,89 @@
-import React from 'react'
-import Dashboard from '../Dashboard'
+import React, { useState, useEffect } from 'react'
+import classNames from 'classnames'
+import { useRouter } from 'next/router'
+import Nav, { navConstants } from '../Dashboard/Nav'
+import { layoutPadding } from '../Dashboard/dashboardConstants'
+import Header from '../Dashboard/Header'
+import useWindowSize from '../../hooks/useWindowSize'
 
 interface Props {
   children: React.ReactNode
+  withPadding?: boolean
 }
 
-const DashboardLayout: React.FC<Props> = ({ children }) => <Dashboard>{children}</Dashboard>
+const DashboardLayout: React.FC<Props> = ({ children, withPadding = true }) => {
```

Comment:
I noticed that the `Dashboard` component was really the `DashboardLayout` component and `DashboardLayout` was a just a wrapper (see line 8 on the left), so I moved it from `Dashboard` to `DashboardLayout`